editor.planx.uk/src/@planx/components/FindProperty/Public/index.tsx

```diff
@@ -111,7 +116,16 @@ function Component(props: Props) {
   }, [data, address]);
 
   const validateAndSubmit = () => {
-    // TODO `if (isValidating)` on either page, wrap Continue button in error mesage?
+    if (isValidating) {
+      setDataFetchError("Please wait for data fetching to complete");
+    }
```

Comment:
After the user has selected an address on either page, we need to make a quick request to Planning Data to fetch supplmental data about it that we can't get from Ordnance Survey - and we want to prevent the user from "continuing" before this fetch completes. 

We used to simply set `isValid={!isValidating}` on the `Card`, but now we want to keep the continue button always enabled.

A couple questions here: 
- What's the best language for explaining this? 
- Is wrapping the "Continue" button in the error right considering the proposed design in #3136 ? If not the button, what else could we wrap? 
  - [TODO] Prevent "Continue" button from expanding to full-screen width when in error state (see screenshot below) 
  
![Screenshot from 2024-05-13 21-02-50](https://github.com/theopensystemslab/planx-new/assets/5132349/c2fb08c4-c35f-47ce-b0af-89ba15fb9eda)

Reply:
Open questions from dev call: 
- Should we move the loading indicator _into_ the button in these instances rather than wrap it in an error?